src/Compilers/VisualBasic/Portable/Semantics/TypeInference/TypeInferenceCollection.vb

```diff
@@ -599,6 +601,33 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End If
         End Sub
 
+        Friend Shared Function MergeTupleNames(first As TypeSymbol, second As TypeSymbol) As TypeSymbol
```

Comment:
The C# version of this method has an additional `target` that seems unnecessary. Consider removing that parameter. Same for `MergeDynamic`.